app/javascript/controllers/calendar_controller.js

```diff
@@ -0,0 +1,28 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ["date"]
+
+  connect() {
+    this.dateTargets.forEach((date) => {
+      date.parentElement.addEventListener("click", this.toggleSelect)
+
+      if (date.hasAttribute("selected")) {
```

Comment:
this renders the blue background on the dates that have been generated through the calculator.